hack/istio/multicluster/install-multi-primary.sh

```diff
@@ -93,8 +97,60 @@ create_remote_secret() {
   fi
 }
 
+MC_EAST_YAML=$(
+cat <<EOF
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: ${SAIL_VERSION}
```

Comment:
Unless we provide a way to specify which istio version you install, leaving this unset will get you the latest supported version which is what we want for these tests.